_layouts/recipe.html

```diff
@@ -8,6 +8,9 @@
 
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta class="swiftype" name="site-id" data-type="integer" content="3" />
+  <meta class="swiftype" name="docs-boost" data-type="integer" content="6" />
+  <meta class="swiftype" name="category" data-type="string" content="recipe" />
```

Comment:
Does it make sense to differentiate `recipe` from `tutorial`?  How is it consumed?

Reply:
I suppose that's more of a question for the content creators? My plan was to distinguish them on the code side, but if a user selects `tutorial` it would include both